### ISSUE TITLE
'math' chapter is hard to follow

### ISSUE THREAD

**Issue by voelzmo** [quii/learn-go-with-tests#457]
Hey there! It seems there's quite some confusion around the math chapter, indicated by #368 and pending PR #266. Some of the points in the original PR seem to be covered already – let me try to put the points which made it hard for me to follow in a list

- [ ] This is the first time you're writing acceptance tests in a `<package_name>_test` package instead of just writing unit tests in the package itself. As mentioned in #266 already, this concept should be introduced with a sentence or two
- [ ] Helper functions, such as `testName` and `simpleTime` are used in tests from both packages, which is never addressed and I ended up just copying them, which felt a bit weird
- [ ] Execution example output suddenly switches from executing `clockface_test.go` to `clockface_acceptance_test.go` for the acceptance test. Maybe it should have been `clockface_acceptance_test.go` from the start?
- [ ] There's a lot of 'commenting out tests' going on, which made it difficult to understand where I'm at currently. Which tests are supposed to pass at which point and when am I commenting them back in?
- [ ] When creating a new test, help the reader understand if it is a unit test or an acceptance test and maybe spell out more explicitly in which file it should go, such that they can get a better feeling for the difference between the two test types
- [ ] section https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/math#write-enough-code-to-make-it-pass-3 uses the wrong circle picture: We use `cos(a)` to compute the `y` coordinate, as explained in the previous section
- [ ] section https://quii.gitbook.io/learn-go-with-tests/go-fundamentals/math#write-enough-code-to-make-it-pass-8 mentions copy-and-pasting `minuteHand` from `secondHand`, but has differences, e.g. not using a function to get the tag string from a point. It's also not mentioned what `minuteHandLength` should be and that the copy-and-pasted function should be called from the `SVGWriter`
- [ ] package names and imports use `github.com/gypsydave5/learn-go-with-tests`
- [ ] some tests use `t.Fatalf` instead of `t.Errorf` without really stating why
- [ ] test output is inconsistent with the previous chapters. Most previous chapters used "got X, want Y", this chapter uses "want Y, got X"
- [ ] link "my final take on this program" points to https://github.com/quii/learn-go-with-tests/tree/9ebe23c4add05e2401255b62dc5e8b59ec751f8b/math/vFinal/clockface/README.md, which doesn't exist

Let me know how you feel about this list and if these points make sense! If you're willing to accept individual PRs for these, I can take some time to tackle them one by one. If you want to take a different approach and/or don't want some of the above items changed, that's fine as well – just lmk.

**Comment by gypsydave5** [comment]
Hey @voelzmo - let me see how far I can get through this list this week.

Better late than never.

**Comment by voelzmo** [comment]
Awesome, thanks @gypsydave5! Let me know if things don't make sense or you have a different view on this. A side note to the part about acceptance tests and using the `<package_name>_test` package for the first time: I think the new chapter on using the `filesystem` API does this already quite nicely, so you might be able to benefit from this without having to write something from scratch here.

**Comment by quii** [comment]
@gypsydave5 did your big PR resolve this?

**Comment by voelzmo** [comment]
Assuming this has been fixed, I'll close this one. Thanks!
